.gitignore

```diff
@@ -4,8 +4,7 @@
 web/.idea
 
 # dependencies
-web/node_modules
```

Comment:
Should leave this. Don't want a Macadamian situation LOL

Reply:
I added "node_modules/" on the root gitignore. That should ignore all the directories name "node_module" in subdirectories.